components/page/index.css

```diff
@@ -12,8 +12,11 @@
  */
 
 :root {
-	background: var(--mod-page-background, var(--spectrum-page-background-color, var(--spectrum-gray-75)));
+	--spectrum-page-background-color: var(--mod-page-background, var(--spectrum-gray-75));
+	--spectrum-page-content-tap-highlight: var(--mod-page-content-tap-highlight, var(--spectrum-transparent-black-25));
```

Comment:
Pulled these variables out of the inline-use to match our standard approach for other components.